zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -200,3 +228,28 @@ class LumiLightAcn014(XiaomiCustomDevice):
             },
         }
     }
+
+
+(
+    QuirkBuilder(AQARA, "lumi.light.acn032")
+    .friendly_name(manufacturer="Aqara", model="Ceiling Light T1M")
+    .adds_endpoint(2, device_type=zha.DeviceType.COLOR_DIMMABLE_LIGHT)
+    .replaces(AqaraLightT1M)
+    .replaces(AqaraLightT1M, endpoint_id=2)
+    .enum(
+        AqaraLightT1M.AttributeDefs.power_on_behavior.name,
+        LumiPowerOnBehaviorMode,
+        AqaraLightT1M.cluster_id,
+        translation_key="power_on_behavior",
+        fallback_name="Power on behavior",
+    )
+    .enum(
+        AqaraLightT1M.AttributeDefs.power_on_behavior.name,
+        LumiPowerOnBehaviorMode,
+        AqaraLightT1M.cluster_id,
+        translation_key="power_on_behavior",
+        fallback_name="Power on behavior",
```

Comment:
Can we rename both to `power_on_state` and "Power on state"? We already have that very similar translation/string defined [here](https://github.com/home-assistant/core/blob/6641cb37998d0c70a9d9a2fef5a7a01cb4126090/homeassistant/components/zha/strings.json#L1176-L1178).

Also note that we'll try to add a suffix in ZHA to duplicate entity names in the future. So, no real need to do it here for now.